mirror-wordpress-plugins

```diff
@@ -145,8 +177,13 @@ class Plugin
     rescue JSON::ParserError => e
       raise WordPressPluginApiError, e.message
     end
-    # Whippet Racetrack error message here was "No plugin info available for <plugin>".
-    raise WordPressPluginApiError, plugin_info["error"].to_s if plugin_info.key?("error")
+    if plugin_info.key?("error")
```

Comment:
If the Api response does not include an error, do we want to remove the `skip-mirror` topic (and any other errors)? I'm assuming it's hypothetically possible that a plugin could be closed and subsequently re-opened, but I don't know for sure if that's the case.

Reply:
I'm in two minds about this, because the automated script won't be able to tell whether `skip-mirror` was added for that reason or whether a develop added the topic manually. Maybe it would be better to add an `active-on-wordpress` topic, then we can search for repos that have both?